src/btree/bt_curnext.c

```diff
@@ -188,15 +188,16 @@ __cursor_var_append_next(
 
         if (cbt->upd_value->type == WT_UPDATE_INVALID) {
             ++*skippedp;
+            cbt->valid_data = true;
```

Comment:
I don't like the naming "valid_data", since we are marking it to be true here, when the update type is invalid. How about "non_deleted_updates".